package.json

```diff
@@ -69,6 +84,7 @@
     "serve": "grunt serve",
     "start": "grunt serve",
     "test-integration": "grunt test-integration",
-    "postinstall": "node test/upgrade-selenium.js && node_modules/protractor/bin/webdriver-manager update"
+    "__postinstall": "node test/upgrade-selenium.js && node_modules/protractor/bin/webdriver-manager update",
+    "_postinstall": "node_modules/protractor/bin/webdriver-manager update"
```

Comment:
This might be causing the Jenkins failure?

Reply:
same as above, putting this back.